src/mvba/consensus.rs

```diff
@@ -252,8 +256,22 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_prevent_decision_conflict() {
+        do_prop_random_msg_delivery(239389896331702023721706851822556999354)
+    }
+
+    #[test]
+    fn test_prevent_double_pre_vote() {
+        do_prop_random_msg_delivery(321606964118541147451444878468141021383)
+    }
```

Comment:
Don't do this, it won't scale. We will continue to modify the random test in the future and that will make this seed meaningless.

Instead duplicate the prop test code and reduce it to a minimal scenario, i.e unravel the loop and hardcode the msg delivery order so that we have a reproducible test case independent of the original prop test.

Reply:
Done.